lib/retriable/exponential_backoff.rb

```diff
@@ -25,19 +25,16 @@ def initialize(opts = {})
 
     def intervals
       intervals = Array.new(tries) do |iteration|
-        [base_interval * multiplier**iteration, max_interval].min
+        [base_interval * multiplier ** iteration, max_interval].min
       end
 
-      return intervals if rand_factor.zero?
```

Comment:
(i get what it does; feel free to keep it NBD i just find that less code = more readable code)